pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestCreateLabel.java

```diff
@@ -38,7 +38,7 @@ public class KitRequestCreateLabel {
     private final String participantCollaboratorId;
     private final String collaboratorIdPrefix;
     private final boolean hasAuth0Token;
-    private final KitRequestSettings kitRequestSettings;
+    private KitRequestSettings kitRequestSettings;
```

Comment:
Had to change it from final so that if it was null in the first try, we can find it directly by the subKit's kit type id and replace it with right values